translations/nl-be.json

```diff
@@ -877,6 +877,7 @@
   "for-proof-print": "Dient voor drukproef",
   "upload-translation-file": "Vertaaldocument uploaden",
   "link-publication": "Koppeling publicatie",
+  "download-documents": "Download documenten",
```

Comment:
Ik denk dat Tanee deze ontbrekende key al heeft gefixt, rechtstreeks op development. (heb net dev gepulled in deze branch)
Dus deze key stond nu dubbel